docker/Dockerfile.pdo-tp-ccf-accvm

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Intel Corporation
```

Comment:
Why do we need a new/separate one compared to the old `Dockerfile.pd-tp-ccf`?   This is more or less a super-set of the other one which still should work with the old automated tests?  Keeping two identical in semantic but different in syntax files is confusing and makes maintainance harder. 

On somewhat related note: this dockerfile (nor the docker-compose) are really azure specific, so i would drop the `acc..` from the names of the files